mobile-workspace/latest/config/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Configure Mobile Workspace
+---
+
+Find Alfresco Mobile Workspace at [Google Play](https://play.google.com/store/apps/details?id=com.alfresco.content.app&hl=en-GB&ah=tbrTPT50zRyrvFT0xxYx0IRH6DA){:target="_blank"} for Android or [Apple App Store](https://apps.apple.com/gb/app/alfresco-mobile-workspace/id1514434480){:target="_blank"} for iOS.
+
+## First screen
+
+From the first screen you enter the URL of your Alfresco Content Services instance into the `Connect to`  field. You will then be asked to authenticate using your Alfresco credentials. If you want to use Single Sign-On (SSO) to connect to the Mobile Workspace you can enter the URL of your Alfresco Identity Management Service. You can enter the URL in the following format `my.alfresco.com`.
+> **Note:** You do not need to enter www because the URL will be formatted by the application.
+
+### Advanced Settings
+
+The advanced settings are preconfigured with the correct default settings to connect to Alfresco. Depending on your installation you may need to change some of these. The advanced settings can be accessed from the first screen. There are a number of options which help to authenticate the application with your Alfresco Content Services instance.
+
+| Field | Derscription |
+| ------- | --------- |
+| HTTPS | Allows you to switch between HTTP or HTTPS. If connecting to Alfresco Content Services Community Edition then HTTP is common but it is generally only used for testing purposes only. |
+| Port | The port used by your Alfresco Content Services instance. If you use HTTPS the default value is `443`, if it is HTTP the default value is `80`. |
+| Path | This is the domain name of your installation /alfresco, e.g. mydomain.com/alfresco. |
```

Comment:
The path to where Alfresco Content Services are located on your domain, defaulting to alfresco as in they are available at mydomain.com/alfresco

Reply:
Done